.github/workflows/makeRelease.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Compile clib4
         run: |
           cd /opt/code && \
-          make -j1 -f GNUmakefile.os4 OS=os4 SHARED=no GITTAG=${{ github.event.release.tag_name }}
```

Comment:
This seems was left since the times that the shared objects were unable to be compiled.

Reply:
It is ok of course.